### PR TITLE
New SFX

### PR DESCRIPTION
**Pull request by codingdesigner**
I was refactoring some effects in my other project and I think this is probably a better home for them. Let me know what you think. thanks

### CODE REVIEW

File: tests/recipes/effect/corners-tucked/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<link rel="stylesheet" href="s.css" />
+<h1>Tests for <a href="../../">recipes</a>/<a href="../">effect</a>/scatter</h1>
```
Comment (MoOx) [inline tests/recipes/effect/corners-tucked/index.html]: There is a mistake in this breadcrumb

File: tests/recipes/effect/scatter/index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<link rel="stylesheet" href="s.css" />
+<h1>Tests for <a href="../../">recipes</a>/<a href="../">effect</a>/scatter</h1>
+
+<p class="doc">
+    Rotate a group of elements as if they were casually tossed on a table
+</p>
+
+<div class="body">
```
Comment (MoOx) [inline tests/recipes/effect/scatter/index.html:9]: Is this wrapper really needed ?

**Comment by MoOx** [comment]
Well done !
Just one thing, I think, even if they share the same code, the two effects under "tucked corners" should be in two distincts mixins (more clear for user). No ?

**Comment by codingdesigner** [comment]
Thanks for the notes. :)

Scatter: I didn't need the wrapper so I removed it. 

Tucked / Taped corners: I reworked them into two mixins that share the same code. I also re-ordered the arguments since I think the color arg will be used more often.

**Comment by MoOx** [comment]
Thanks !

**Pull request merged**